dev/tasks/tasks.yml

```diff
@@ -2160,3 +2227,184 @@ tasks:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "macOS"
+      artifact: "source"
+      flag: "TEST_RUBY=1"
+
+  verify-rc-source-macos-python:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "macOS"
+      artifact: "source"
+      flag: "TEST_PYTHON=1"
+
+  verify-rc-source-macos-js:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "macOS"
+      artifact: "source"
+      flag: "TEST_JS=1"
+
+  verify-rc-source-macos-go:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "macOS"
+      artifact: "source"
+      flag: "TEST_GO=1"
+
+  verify-rc-source-macos-rust:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "macOS"
+      artifact: "source"
+      flag: "TEST_RUST=1"
+
+  verify-rc-source-macos-integration:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "macOS"
+      artifact: "source"
+      flag: "TEST_INTEGRATION=1"
+
+  verify-rc-source-linux-java:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "ubuntu"
+      artifact: "source"
+      flag: "TEST_JAVA=1"
+
+  verify-rc-source-linux-csharp:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "ubuntu"
+      artifact: "source"
+      flag: "TEST_CSHARP=1"
+
+  verify-rc-source-linux-ruby:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "ubuntu"
+      artifact: "source"
+      flag: "TEST_RUBY=1"
+
+  verify-rc-source-linux-python:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "ubuntu"
+      artifact: "source"
+      flag: "TEST_PYTHON=1"
+
+  verify-rc-source-linux-js:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "ubuntu"
+      artifact: "source"
+      flag: "TEST_JS=1"
+
+  verify-rc-source-linux-go:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "ubuntu"
+      artifact: "source"
+      flag: "TEST_GO=1"
+
+  verify-rc-source-linux-rust:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "ubuntu"
+      artifact: "source"
+      flag: "TEST_RUST=1"
+
+  verify-rc-source-linux-integration:
+    ci: github
+    platform: linux
+    template: verify-rc/github.nix.yml
+    params:
+      os: "ubuntu"
+      artifact: "source"
+      flag: "TEST_INTEGRATION=1"
+
+  verify-rc-source-windows:
+    ci: github
+    platform: linux
+    template: verify-rc/github.windows.source.yml
+
+  verify-rc-wheels-windows:
+    ci: github
+    platform: linux
+    template: verify-rc/github.windows.wheels.yml
```

Comment:
The copypasta is a bummer. At some point we might look at some way to condense these specifications and reuse templates (somehow), but not worth the effort right now

Reply:
Agreed.